aten/src/THCS/generic/THCSTensor.cu

```diff
@@ -19,6 +19,16 @@
 #define I_INFO(tensor) getTensorInfo<THCIndexTensor, uint64_t>(state, tensor)
 #define V_INFO(tensor) getTensorInfo<THCTensor, uint64_t>(state, tensor)
 
+THCudaIntTensor *THCSTensor_(toCSR)(THCState *state, THCIndexTensor *rowIndices, int64_t dim, int64_t nnz) {
+  THCudaIntTensor *csr = THCudaIntTensor_newWithSize1d(state, dim + 1);
+  THCudaIntTensor *rowIndicesInt = THCudaIntTensor_newWithSize1d(state, rowIndices->size[0]);
```

Comment:
In general indices can have much more than a single dim, this should probably be checked here, right?

Reply:
Yes, I will add a check.